modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/js/edit_blueprint/PreviewSidebar.js

```diff
@@ -175,6 +176,19 @@ function PreviewSidebar({
 				</div>
 			</nav>
 
+			{results.warnings &&
+				results.warnings.length &&
+				results.warnings.map((warning, idx) => (
```

Comment:
Let's use the full word `index`. It's more clear as it could be confused with `id`. If we need to abbreviate, I'd rather use `i`.